dspot/src/main/java/fr/inria/diversify/utils/AmplificationHelper.java

```diff
@@ -290,7 +319,8 @@ public static String getClassPath(DSpotCompiler compiler, InputConfiguration con
             final Long average = average(valuesToMethod.keySet());
             while (reducedTests.size() < MAX_NUMBER_OF_TESTS) {
                 final Long furthest = furthest(valuesToMethod.keySet(), average);
-                reducedTests.add(valuesToMethod.get(furthest).get(0));
+                CtMethod<?> discardedTest = valuesToMethod.get(furthest).get(0);
+                reducedTests.add(discardedTest);
```

Comment:
The name is wrongly chosen. In the opposite that happen: we keep the "furthest" method. That is why I removed your call to `remove()` on the map of parents. I think we can inline the variable here.

Reply:
I have to be more focused. Sorry

Reply:
No you good, That's the whole point of the review process :)